src/Protocol/Diaspora.php

```diff
@@ -449,14 +453,14 @@ public static function decode(string $xml, string $privKey = '')
 	/**
 	 * Dispatches public messages and find the fitting receivers
 	 *
-	 * @param array $msg     The post that will be dispatched
-	 * @param bool  $fetched The message had been fetched (default "false")
+	 * @param array $msg       The post that will be dispatched
+	 * @param int   $direction Indicates if the message had been fetched or pushed
 	 *
 	 * @return int The message id of the generated message, "true" or "false" if there was an error
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 * @throws \ImagickException
 	 */
-	public static function dispatchPublic($msg, bool $fetched = false)
+	public static function dispatchPublic($msg, int $direction)
```

Comment:
Please add a reference to the constants the parameter is taking its values from.